django/db/backends/mysql/base.py

```diff
@@ -272,7 +273,7 @@ def init_connection_state(self):
             with self.cursor() as cursor:
                 cursor.execute('SET ' + ', '.join(assignments))
 
-    def create_cursor(self, name=None):
+    def create_cursor(self, name=None, chunk_size=constants.GET_ITERATOR_CHUNK_SIZE):
```

Comment:
Is there any point in `chunk_size=constants.GET_ITERATOR_CHUNK_SIZE` (rather than say, `None`) where it's unused?